asteroid/masknn/attention.py

```diff
@@ -45,12 +45,6 @@ def __init__(
 
         self.mha = MultiheadAttention(embed_dim, n_heads, dropout=dropout)
         self.dropout = nn.Dropout(dropout)
-        self.pos_wise_ff = nn.Sequential(
```

Comment:
Why did you decide to remove it?

Reply:
Because I got confused it is not in DPTNet sorry I confused also you. 
Also we can implement the whole with two permutations. This is not efficient